### ISSUE TITLE
Bug: Check mode doesn't exit with code 1

### ISSUE THREAD

**Issue by andys8** [lspitzner/brittany#353]
## Issue

The `--check-mode` feature is supposed to exit with exit code 1 if there are changes. I doesn't seem to do this and instead return exit code 0 (success).

```shell
brittany --check-mode src/Bot/Dsl/Modules.hs
ERROR: brittany pretty printer returned syntactically invalid result.
formatting would modify: src/Bot/Dsl/Modules.hs

echo $?
0
```

```
  -c,--check-mode     check for changes but do not write them out
                      exits with code 0 if no changes necessary, 1 otherwise
                      and print file path(s) of files that have changes to stdout
```

## Code

https://github.com/lspitzner/brittany/blob/4d064db674203626fe5011d10874fcbc335ec9b1/src/Language/Haskell/Brittany/Main.hs#L464-L466

**Comment by tfausak** [comment]
I am not able to reproduce this behavior. Which version are you using?

``` sh
$ cabal exec -- brittany --version
brittany version 0.14.0.0
Copyright (C) 2016-2019 Lennart Spitzner
Copyright (C) 2019 PRODA LTD
There is NO WARRANTY, to the extent permitted by law.

$ cat Example.hs
x=()

$ cabal exec -- brittany --check-mode Example.hs
formatting would modify: Example.hs

$ echo $?
1
```

**Comment by andys8** [comment]
Hey, thanks for looking into it.

I retested the issue with version `0.13.1.2`, and I can *not* reproduce the issue either.

```shell
brittany --check-mode src/Bot/Dsl/Modules.hs
formatting would modify: src/Bot/Dsl/Modules.hs

echo $?
1
```

I should have done a better job when creating the ticket and add more information. So, either the issue was not the case to begin with (which would at least surprise me, because I did create a workaround script and an issue here), or was fixed or influenced due to other changes over time.

Thanks for looking into it, but I would keep it here and close the issue :)
